### PR TITLE
commit

### PR DESCRIPTION
**Pull request by anastasia-radchencko**
https://anastasia-radchencko.github.io/react_people-table-advanced/

### CODE REVIEW

File: src/components/PeopleFilters.tsx
Comment (etojeDenys) [inline src/components/PeopleFilters.tsx:12]: move all query param names to some enum it will help us to easily manipulate them

File: src/components/PeopleFilters.tsx
Comment (etojeDenys) [inline src/components/PeopleFilters.tsx:14]: we can move this array out of the component as constant

**Review by etojeDenys — CHANGES_REQUESTED** [review]

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Great job overall! Just one tiny thing needs to be fixed before approval)
check out your sort button
https://www.loom.com/share/408ad4cdfac6455588938d93da78aa52

**Pull request closed without merging**
